src/api/handlers/charts/charts_test.go

```diff
@@ -145,6 +164,27 @@ func TestSearchCharts200(t *testing.T) {
 	assert.Equal(t, len(helpers.MakeChartResources(charts)), len(httpBody.Data), "number of charts returned")
 }
 
+func TestSearchCharts404(t *testing.T) {
+	w := httptest.NewRecorder()
+	params := chartsapi.SearchChartsParams{
+		Name: "drupal",
+	}
+	chImplementation := mocks.NewMockCharts(mocks.MockedMethods{
+		Search: func(params chartsapi.SearchChartsParams) ([]*models.ChartPackage, error) {
+			var ret []*models.ChartPackage
+			return ret, errors.New("error searching charts")
+		},
+	})
+	resp := SearchCharts(params, chImplementation)
+	assert.NotNil(t, resp, "SearchCharts response")
+	resp.WriteResponse(w, runtime.JSONProducer())
+	assert.Equal(t, w.Code, http.StatusBadRequest, "expect a 400 response code")
+	var httpBody models.Error
+	assert.NoErr(t, testutil.ErrorModelFromJSON(w.Body, &httpBody))
+	assert.Equal(t, *httpBody.Code, int64(400), "response code in HTTP body data")
```

Comment:
is this `int64(..)` really needed for readability? Constant are automatically interpreted to be of the required type.